MAVProxy/tools/MAVExplorer.py

```diff
@@ -625,6 +625,34 @@ def cmd_param(args):
         if fnmatch.fnmatch(str(p).upper(), wildcard.upper()):
             print("%-16.16s %f" % (str(p), mestate.mlog.params[p]))
 
+def cmd_paramchange(args):
+    '''show param changes'''
+    types = set(['PARM','PARAM_VALUE'])
+    vmap = {}
+    while True:
+        m = mestate.mlog.recv_match(type=types, condition=mestate.settings.condition)
+        if m is None:
+            break
+        if m.get_type() == 'PARM':
+            pname = m.Name
+            pvalue = m.Value
+        elif m.get_type() == 'PARAM_VALUE':
+            pname = m.param_id
+            pvalue = m.param_value
+        else:
+            continue
+        if pname.startswith('STAT_'):
```

Comment:
We have a list of differently-uninteresting parameters here: https://github.com/ArduPilot/pymavlink/master/mavparm.py#L14 - there might be a few more to add here.

Reply:
I really just want to filter out ones that change frequently in flight. STAT_ is good for that